app/assets/javascripts/app/controllers/cti.coffee

```diff
@@ -1,6 +1,7 @@
 class App.CTI extends App.Controller
   events:
-    'click .js-check': 'done'
+    'click .js-check': 'done',
+    'click [data-type=new]': 'new_user'
```

Comment:
Please use 'js-' prefixed classes (like in the line above) to identify elements that take action(s) in JS instead of data attributes.